packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -213,6 +213,8 @@ export default abstract class BaseCall implements IWebRTCCall {
       const bye = new Bye({
         sessid: this.session.sessionid,
         dialogParams: this.options,
+        cause: 'USER_BUSY',
+        causeCode: 17,
```

Comment:
Can you explain how you found this specific cause type and code?

Reply:
@ryan tells me to use this cause and code, you can see all the conversation in the Slack channel.

Reply:
@ryan tells me if you reject the call, you need to indicate the cause to USER_BUSY when you invoke telnyx_rtc.bye.